source/testnet/references/id-accounts.rst

```diff
@@ -23,13 +19,13 @@ Identities and accounts
 
 
 In order to be an active participant on the Concordium blockchain (e.g., hold,
-send, receive `GTU`_) a user must create an *account*.
+send, receive :ref:`glossary-GTU`) a user must create an *account*.
 
 To facilitate compliance with relevant regulations, an account can only be
-created from an *identity* which is issued `off-chain`_ by an Identity provider.
+created from an *identity* which is issued :ref:`glossary-off-chain` by an Identity provider.
 While an account has to be created from an identity, the user's privacy is still
 protected, and the account owner's identity can only be revealed via the process
-of *anonymity revocation*, which can only happen under stringent regulations,
+of `anonymity revocation`_, which can only happen under stringent regulations,
 such as upon a court order. In particular, a key feature of the design of
 identities and accounts is that the identity provider cannot reveal the identity
 of an account on their own.
```

Comment:
This needs some rework due to initial accounts. The identity provider can tell who they belong to, since it creates them (in essence).

Reply:
Yeah, that makes sense. I'll try reworking it.

Reply:
I tried reworking this bit, but I'm not sure if it's sufficient. If not, I might need some back-up from someone who knows more about it.